packages/vm/src/runTx.ts

```diff
@@ -210,6 +218,20 @@ async function _runTx(vm: VM, opts: RunTxOpts): Promise<RunTxResult> {
     }
     stateAccesses = vm.evm.verkleAccessWitness
     txAccesses = new VerkleAccessWitness({ verkleCrypto: vm.stateManager.verkleCrypto })
+  } else if (vm.common.isActivatedEIP(7864)) {
+    if (vm.evm.binaryTreeAccessWitness === undefined) {
+      throw Error(`Binary tree access witness needed for execution of binary tree blocks`)
+    }
+
+    if (!(vm.stateManager instanceof StatefulBinaryTreeStateManager)) {
+      throw EthereumJSErrorWithoutCode(
+        `Binary tree State Manager needed for execution of binary tree blocks`,
+      )
+    }
+    stateAccesses = vm.evm.binaryTreeAccessWitness
+    txAccesses = new BinaryTreeAccessWitness({
+      hashFunction: (vm.evm.binaryTreeAccessWitness as BinaryTreeAccessWitness).hashFunction,
```

Comment:
Couldn't this already be typed as BinarytreeAccessWItness fro the EVM? Or do we have to use AccessWitnessInterface?